tools/gulptasks/scripts-messages.js

```diff
@@ -33,14 +33,15 @@ const htmlEscapeTable = {
         replacement: '&gt;'
     }
 };
+const parseMarkdownCRLF = /\r\n|\r/;
 const parseMarkdownBlockCode = /(?:^|\r\n|\n|\r)```(\w*)([\s\S]+?)[\n\r]+```/;
 const parseMarkdownCode = /`([^`\s](?:[^`]|\s)*?)`/;
 const parseMarkdownHeadline = /(?:^|\r\n|\n|\r)(#{1,5})([^\n\r]*)\1?/;
 const parseMarkdownFormat = /(?:^| )(\*{1,3})(\w(?:[^\*]| )*?)\1/;
 const parseMarkdownLink = /\[([^\]]+?)\]\(((?:[^\)]|\s)+?)\)/;
 const parseMarkdownList =
     /(?:^|\r\n|\n|\r)[\t ]*[\-\+\*][\t ]+([\s\S]*?)(?=(?:\r\n|\n|\r)+|$)/;
-const parseMarkdownParagraphs = / {2,}(?:\r\n|\n|\r)| ?(?:\r\n|\n|\r){2,}/;
+const parseMarkdownParagraphs = /(\n){2,}/;
```

Comment:
`/  \n{1,}|\n{2,}/` // note: two space char at the beginning

Reply:
Replaced with `/ {2,}(?:\n)| ?(?:\n){2,}/`, the above one produces empty paragraphs